css/_uswds-theme-custom-styles.scss

```diff
@@ -80,6 +80,10 @@ $site-background-color: 'gray-2';
 
 body {
   @include u-bg($site-background-color);
+
+  &.layout-demo {
+    @include u-bg('white');
+  }
```

Comment:
How about just adding the gray on `.default-container` and remove the override?

Reply:
OK, done